05-services.md

```diff
@@ -398,7 +399,8 @@ services:
   web:
     build: .
     depends_on:
-      db:
+      database:
+        service: db
```

Comment:
I think only having this example might be a bit confusing for new users who don't know about this feature. I'd leave this example as is, and add a specific example about the `service` key under it's explanation